docs/mkcloud.md

```diff
@@ -47,13 +47,26 @@ It requires SLE-12 or openSUSE Leap 42.2 or newer as host OS.
     ```
     $ sudo chkconfig libvirtd on
     ```
+* /etc/lvm/lvm.conf also needs to be edited to filter mkcloud logical volumes.
+
+  * Add the following regular expressions to the filter value in the devices
+    { } block before "a/.*/":
+
+    "r|/dev/mapper/cloud-|", "r|/dev/cloud/|", "r|/dev/disk/by-id/|"
+
+    The filter should also include something like "r|/dev/dm-|" or
+    "r|/dev/dm-1[56]|", but the exact values depend on your local
+    system setup and could change over time or have side-effects
+    (on lvm in dm-crypt or lvm in lvm), so please add/modify it manually.
+
 
 ### mkcloud
 
 To use mkcloud the following additional steps are needed:
 
 * Create a disk file where the virtual machines are going to be stored. The
-  minimum recommended is 80 GB.
+  minimum recommended is 80 GB. A cloud created using the plain option will
+  require 102 GB.
```

Comment:
Maybe in your case its 102GB. But a minimum should still fit into 80GB.
And btw. this step is not "needed" in any case. Its only "needed" if the host does not have an LVM with enough space left (which is recommended anyway).
Creating a sparse file for an LVM is just a workaround for laptops or other small devices that can not be changed quickly to have an LVM or separate disk for cloud development (as recommended).